### DISCUSSION TITLE
Should we prevent scrolling when feedback modal is open?

### DISCUSSION THREAD

**Discussion by mingjunlu** [withastro/docs#5334] [Ideas]
Hi,
I noticed that when the feedback modal is open, the user can still scroll the page.

<details>
<summary>Here's a video clip</summary>

https://github.com/withastro/docs/assets/40516784/0c130ae7-c4f7-46e9-b958-40994f37f17e

</details>

Typically when a fullscreen overlay appears, the page would be "locked" for a better user experience.
Since the search modal prevents scrolling, I wonder whether we should do the same with the feedback modal 🤔

If so, I'm willing to submit a PR for this 😊

**Comment by sarah11918** [comment]
Would like to see people contribute here to say whether this is a problem. I actually LIKE the current behaviour and would not want the scroll to be limited. (I always prefer the ability to scroll in the background and do not like overlays to hijack this behaviour). But, maybe I'm missing something, or there's an accessibility issue I don't know about! So, happy for others to say why this might be a problem to fix!

**Comment by ArmandPhilippot** [comment]
We're closing stale and outdated discussion to keep only the discussions still relevant to the current project version and/or the current docs state.

If you think this is a mistake and that this discussion should remain open, please feel free to reopen.

Thank your for your contribution!
